atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/TransactionCacheValueStoreImpl.java

```diff
@@ -76,6 +74,28 @@ public void cacheEmptyReads(TableReference table, Set<Cell> emptyCells) {
         }
     }
 
+    @Override
+    public TransactionCacheValueStore createWithFilteredSnapshot(CommitUpdate commitUpdate) {
```

Comment:
Definitely, or else we'd have a _bad time_ internally (we were conflicting when we wrote this the first time because of the awkwardness of the `LeasedLockToken`, but those issues have been flushed out.